docs/getstarted/telemetry.md

```diff
@@ -124,15 +124,6 @@ Some precomputed recommendations are shipped as part of the product while additi
 
 When you open a file type for which VS Code does not have any precomputed recommendation, it asks the Extension Marketplace for extensions that declare that they support this file type. If the query returns extensions you don't have installed, VS Code will provide a notification.
 
-## For extension authors
-
-If you have created a VS Code extension, you can use the VS Code telemetry infrastructure for reporting through the [vscode-extension-telemetry](https://www.npmjs.com/package/vscode-extension-telemetry) npm module. This module provides a consistent way for extensions to report telemetry over [Azure Monitor and Application Insights](https://azure.microsoft.com/services/monitor/). The module respects the user's decision about whether or not to send telemetry data via the `telemetry.telemetryLevel` setting. Additionally, this module guarantees backwards compatability against previous versions of VS Code.
```

Comment:
For discoverability, do we want to leave the H2 header and then add some general text and a link to the new topic?
This would also help anyone who had bookmarked this section. We don't usually add redirects for header links.

Reply:
Yes this makes sense, let me do that.

Reply:
This link can just be (/api/extension-guides/telemetry.md)

Reply:
Right. Thanks for catching that. I pushed a fix.